docs/components/store.md

```diff
@@ -94,6 +94,12 @@ Flags:
                                  --no-cache-index-header option is specified.
       --enable-auto-gomemlimit   Enable go runtime to automatically limit memory
                                  consumption.
+      --estimated-series-size-stat=max
```

Comment:
Same question as in the other PR: how does one know which choice is the "correct" one?

Reply:
![image](https://github.com/user-attachments/assets/f33b13c1-6d98-4d07-aeb5-0f9744cc7d32)

We expose metrics about how much lazy posting optimized and how much we overfetch for series. If users want to optimize lazy posting more then they can use a smaller percentile stat.

It is hard to say `correct` or not. It is more like how to tune lazy posting to make it optimize more scenarios. In general, a block with 20M series, it is P9999 size should be only smaller than 2000 series. It is probably more like how much I am able to tolerate if things go wrong. And users can play with this config and the metrics above.